service/javadsl/kafka/server/src/main/scala/com/lightbend/lagom/internal/javadsl/broker/kafka/JavadslRegisterTopicProducers.scala

```diff
@@ -100,7 +100,7 @@ class JavadslRegisterTopicProducers @Inject() (
 
               case other =>
                 log.warn {
-                  s"Unknown topic producer ${other.getClass.getName}. " +
+                  s"Expected Topic type ${classOf[MethodTopicHolder].getName}, but found incompatible type ${other.getClass.getName}." +
```

Comment:
@Munandermaan, here instead we can check for `null` and have a specific error. For instance, 

```scala
case null =>  log.error(s"Expected Topic type ${classOf[MethodTopicHolder].getName}, but null instance was passed", new NullPointerException())
```

Something like that

Reply:
Thanks @renatocaval Now more clear about it and done required changes